test-suite/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
```

Comment:
do we need a whole new test suite for this? Could we not have added to the already existing tests

Reply:
done via https://github.com/micronaut-projects/micronaut-serialization/pull/492